data-prepper-plugins/otel-trace-raw-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/oteltrace/OTelTraceRawProcessor.java

```diff
@@ -205,7 +205,7 @@ private List<Span> getTracesToFlushByGarbageCollection() {
                             } else {
                                 spans.forEach(span -> {
                                     recordsToFlush.add(span);
-                                    LOG.warn("Missing trace group for SpanId: {}", span.getSpanId());
+                                    LOG.debug("Missing trace group for SpanId: {}", span.getSpanId());
```

Comment:
Why make this a debug and not a warning? 

@chenqi0805 , Thoughts on this change?

Reply:
This is happening in a loop.

It might make sense to have the following:

```
} else {
    LOG.warn("There are {} spans with missing trace groups. Unable to populate with trace group information.", spans.size());
    spans.forEach(span -> {
        recordsToFlush.add(span);
        LOG.debug("Missing trace group for SpanId: {}", span.getSpanId());
    });
}
```

Reply:
This change sounds good. Updated with recommended changes.